src/Ccap/Codegen/Parser.purs

```diff
@@ -27,16 +27,14 @@ tokenParser :: GenTokenParser String Identity
 tokenParser = makeTokenParser $
   LanguageDef (unGenLanguageDef javaStyle)
     { reservedNames =
-        [ "module"
-        , "type"
-        , "boolean"
-        , "date"
-        , "dateTime"
+        [ "boolean"
         , "decimal"
         , "int"
+        , "module"
         , "optional"
         , "string"
-        , "time"
+        , "type"
+        , "wrap"
```

Comment:
I don't think most of these really need to be reserved names, but I do agree it seems odd to use the `reserved` parser for matching non-reserved names. Since the `reserved` parser also matches non-reserved names it seems like a bit of a misomer.

Anyway, we can figure out how we want to handle that later.